Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "binutils-2.28.1.tar.xz" "binutils" "host-binutils" "const-decl-fix" "\
        libiberty/strerror.c \
    ";


===============================================================================
--- a/libiberty/strerror.c	2017-07-04 10:43:21.000000000 +0200
+++ b/libiberty/strerror.c	2018-01-04 13:37:18.000000000 +0100
@@ -471,10 +471,10 @@
 
 
 #ifndef sys_nerr
-extern int sys_nerr;
+extern __const int sys_nerr;
 #endif
 #ifndef sys_errlist
-extern char *sys_errlist[];
+extern __const char *__const sys_errlist[];
 #endif
 
 #endif
